### ISSUE TITLE
Comments not working with kamaleon theme

### ISSUE THREAD

**Issue by helhamma** [open-classifieds/OpenClassifieds#16]
Hi,

I can't make disqus comments working with Kamaleon theme while it works fine with wpclassifieds theme for example...  is there anything to do to fix it ?

Software version: v1.8.3

Link to your site/problem: http://marcel.net84.net/PRO_OC_183_customized/content/item.php?item=1&type=offer&category=tv&title=panasonic-p42v10

What steps will reproduce the problem?
1. select Kamaleon theme (admin/settings.php, look & feel section)
2. enter your disqus account name in comments field (admin/settings.php, external tools section)
3. save settings
4. display any ad from your website

What is the expected output?
a comments section should be displayed at bottom page

What do you see instead? 
no comment section (a strange "comments " hyperlink is displayed under Ad location)

**Comment by neo22s** [comment]
I tihnk is better you use our pro forum for such problem.

regards

**Comment by helhamma** [comment]
Hi neo22s,

Could you give me is pro forum url please?

Thanks!

**Comment by neo22s** [comment]
It's an email address we sent to you when you bought the pro version...check the email we sent to you with the download link.

regards

**Comment by emanwebdev** [comment]
I improved the discus inclusion in previous version already 
(basically the ability to include options and customizations)

but I don't have this Kamaleon theme to test with it

**Comment by neo22s** [comment]
Should work on any theme anyway, I could send to you all the pro themes, but they cant be included in the repo as you may understand. regards!

**Comment by emanwebdev** [comment]
Hola
I improved the discus but needs to rewrite a bit the templates

This is due to the lack of an abstraction layer in OC

There are unfortunately no "helpers" or GUI primitives 
and it's why ALL the themes need to be rewritten ....

**Comment by emanwebdev** [comment]
> I could send to you all the pro themes, but they cant be included in the repo as you may understand 

I just see you offer a new "Lite" package
I don't have a website running OC so far but when I'm finished
merging my new version with your new version, I will run with v. 1.8.4+

**Comment by neo22s** [comment]
I know of these issues that's why I started with https://github.com/neo22s/openclassifieds2

But not much free time left...sadly...

**Comment by emanwebdev** [comment]
unless we write a v.2.0 ?
with Autoloading Classes

**Comment by emanwebdev** [comment]
oooppps..  :)  funny. same thought @ same time.....

**Comment by emanwebdev** [comment]
> I started with https://github.com/neo22s/openclassifieds2

can download to try?

**Comment by neo22s** [comment]
of course!

it's just that is a bit out of dated, the kohana version is not the newest either the twitter boostrap, and many things to do. 

I started it 1 year a go: https://www.pivotaltracker.com/projects/331093#

regards

**Comment by neo22s** [comment]
december 2011 I started. we can talk about it if you are interested of course!

**Comment by emanwebdev** [comment]
> I started with https://github.com/neo22s/openclassifieds2

Oh. MVC. Kohanna. i see you already defined routes aso
Only memcache
all Views to do

**Comment by emanwebdev** [comment]
> december 2011 I started. we can talk about it if you are interested of course!

Are you in Spain??? (it's half an hour past midnight now here in Switzerland)

or in Brazil or elsewhere???

I cannot speak now, too late to make noise, but yes, let's Skype 
during daytime

**Comment by emanwebdev** [comment]
I will buy your "Lite" package now to test and really start with responsive templates

**Comment by emanwebdev** [comment]
> I started with https://github.com/neo22s/openclassifieds2

you say 2011, but one can read "7 days ago" on GitHub
maybe when you created that repo??

**Comment by neo22s** [comment]
I am today at SFO, but tuesday at Barcelona.

Don't buy the lite, I can send it to you for free for sure. Just do not share it :)

I created the repo few days a go from SVN to GIT since people seems to like it more...

**Comment by emanwebdev** [comment]
> I created the repo few days a go from SVN to GIT since people seems to like it more...

right. U know, GitHub has been written by Linus Torvalds himself for the Linux kernel development.
He wanted something very open, "social" and very fast

right now (5 AM...) playing with GitGUI
I merged my version 1.8.2 (fixed some bugs + improvements) with the new 1.8.3

**Comment by emanwebdev** [comment]
But I still have to finish the merge of 1.7.7+ with your new 1.8.3

And it's a BIG task....

**Comment by neo22s** [comment]
man! go to sleep! hehe

Please detail as much as possible the changes so would be easier to review.

Thanks for the job!

**Comment by emanwebdev** [comment]
Right, You have a pretty detailed changelog. with the list of modified files.

I will keep such a list from now. I agree it's important and fair/clean

Right. Gotta sleep

Thanks Chema. L8ter
